src/Components/Preloaded/Preload.tsx

```diff
@@ -0,0 +1,13 @@
+import { FC } from "react";
+import { Spin } from "antd";
+import "antd/dist/antd.css";
+
+const Preloaded: FC = () => {
```

Comment:
better to name this component and folder as Preloader

Reply:
done